TestFoundation/TestFileHandle.swift

```diff
@@ -8,6 +8,7 @@
 //
 
 import Dispatch
+@testable import Foundation
```

Comment:
This must be guarded with `NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT`; see just above.